### PR TITLE
Implement sending location via longitude and latitude.

### PR DESCRIPTION
**Pull request by gmoshkin**
I found myself in the need of a way to send locations from a command line and thought it might be useful for someone else.

### CODE REVIEW

File: telegram_send.py [hunk 1]
```diff
@@ -209,6 +212,16 @@ def send_message(message):
...
 
+    if locations:
+        it = iter(locations)
+        for loc in it:
+            if ',' in loc:
+                lat, lon = loc.split(',')
+            else:
+                lat = loc
+                lon = next(it)
+            bot.send_location(chat_id=chat_id, latitude=float(lat), longitude=(lon))
```
Comment (rahiel) [inline telegram_send.py]: `longitude=(lon)`, should that be `longitude=float(lon)`?

Comment (gmoshkin) [reply telegram_send.py]: yep, sorry about that

**Comment by rahiel** [comment]
Thanks for your contribution @gmoshkin!

**Comment by rahiel** [comment]
This feature is now released in version 0.20

**Pull request merged**